test/e2e/util.go

```diff
@@ -403,14 +404,13 @@ func TearDown(namespace string) {
 		return client.DeleteAllOf(clientCtx, &operatorsv1alpha1.Subscription{}, inNamespace)
 	}).Should(Succeed(), "failed to delete test subscriptions")
 
-	Eventually(func() (remaining []operatorsv1alpha1.Subscription, err error) {
-		list := &operatorsv1alpha1.SubscriptionList{}
-		err = client.List(clientCtx, list, inNamespace)
-		if list != nil {
-			remaining = list.Items
+	Eventually(func() ([]unstructured.Unstructured, error) {
+		var subscriptiongvr = schema.GroupVersionResource{Group: "operators.coreos.com", Version: "v1alpha1", Resource: "subscriptions"}
```

Comment:
nit: could we move this assignment out of the `Eventually()` into the var block above since it only needs to be done once (and is not part of the core polling logic)?

Reply:
Sure.  I'll move it.  Thanks!